ansible/roles/sysctl/defaults/main.yml

```diff
@@ -105,6 +105,16 @@ sysctl__hardening_ipv6_disabled: False
 sysctl__hardening_experimental_enabled: False
                                                                    # ]]]
                                                                    # ]]]
+
+# Performance parameters [[[
+# ---------------------
+# .. envvar:: sysctl__hardening_enabled [[[
+#
+# Should the TCP performance options be applied?
+sysctl__tcp_performance_enabled: False
+
+                                                                   # ]]]
+                                                                   # ]]]
```

Comment:
The additional markers will mess up folding in `vim`. Remove one marker from the previous variable to keep them balanced. Two markers should not have a blank space before them, so that spacing of variables is consistent.

Reply:
I'm confused. Folding is correct in my vim because of the new section "Performance parameters"

Reply:
Right, I see a new section now. Good point.